src/sass/_common/base/constants.scss

```diff
@@ -36,6 +36,7 @@ $COLOR_LIGHT_GRAY    : $fillColor;     // #f2f2f2
 $COLOR_GRAY          : $borderColor;   // #dedede
 $COLOR_DARK_GRAY     : #44484c;
 $COLOR_GREEN         : $positiveColor; // #2e8836
+$COLOR_LETTER_GRAY   : #333;
```

Comment:
Probably meant to put in `src/sass/app_2/_common/base/constants.scss` as it's going to be used in app_2 only.

Reply:
got it thanks :)